int16 rpm
int16 mph
float32 kmh
float32 total_driven_centimeters